### PR TITLE
Unified all courses follow-up

### PR DESCRIPTION
**Pull request by trevorjonesd2l**
Pretty much all just PR feedback from last time. No tests yet, another PR.

### CODE REVIEW

File: src/d2l-all-courses-unified-content.html [hunk 1]
```diff
@@ -113,10 +88,30 @@
...
+			},
+
+			_updateTileSizes: function() {
+				this._rescaleCourseTileRegions();
+				this._tileSizes = Math.floor(100 / this._numColsOverlay) + 'vw';
+			},
+			_enrollmentsChanged: function() {
+				this._noCoursesInSearch = false;
+				this._noCoursesInSelection = false;
+				if (!(this.filteredEnrollments.length > 0)) {
```
Comment (ryantmer) [inline src/d2l-all-courses-unified-content.html]: Could ditch the `!` here, to just `if (this.filteredEnrollments.length === 0)` (similarly below)

File: lang/en.html [hunk 2]
```diff
@@ -74,9 +74,8 @@
 			'sorting.datePinned': 'Date Pinned',
 			'sorting.department': 'Department',
 			'sorting.lastAccessed': 'Last Accessed',
-			'sorting.enrollmentDate': 'Enrollment Date',
-			'sorting.current': 'Current',
```
Comment (ryantmer) [inline lang/en.html]: Why no more current sort? We're just using "Default" in that case, ja?

File: src/d2l-all-courses.html [hunk 3]
```diff
@@ -77,12 +77,12 @@
 										<div class="dropdown-content-header">
 											<span>{{localize('sorting.sortBy')}}</span>
 										</div>
-										<d2l-menu-item-radio hidden$="[[!updatedSortLogic]]" class="dropdown-content-gradient" value="Default" text="{{localize('sorting.default')}}"></d2l-menu-item-radio>
-										<d2l-menu-item-radio value="OrgUnitName" class$="{{_showDropdownGradient(updatedSortLogic)}}" text="{{localize('sorting.sortCourseName')}}"></d2l-menu-item-radio>
+										<d2l-menu-item-radio hidden$="[[!updatedSortLogic]]" class="dropdown-content-gradient" value="Default" text="{{localize('sorting.sortDefault')}}"></d2l-menu-item-radio>
```
Comment (ryantmer) [inline src/d2l-all-courses.html]: nbd, but these `localize`s can also be one-way bindings

File: lang/en.html (hunk 2 above)
Comment (trevorjonesd2l) [reply lang/en.html]: ja

**Pull request merged**